### DISCUSSION TITLE
Contact the developers

### DISCUSSION THREAD

**Discussion by ktechhydle** [ktechhydle/mprun_repo#5] [General]
# A place to get answers and more
Welcome to the discussion. This thread is for contacting the developers. If you have a feature request or bug report, please do it [here](https://github.com/ktechhydle/mprun_repo/issues/new/choose). 

> [!NOTE]
> If you are trying to get help using MPRUN, please use our dedicated [subreddit](https://www.reddit.com/r/mp_software/).

**Comment by tristanvandevenne** [comment]
Got a few questions for experienced dev that work in AI images and video generation of girls

This post in only for experienced devs that know how to do the following:
Create workflows on Comfy Ui / foocus
Put it serverless on[ https://tensordock.com/](https://tensordock.com/) or[ https://www.runpod.io/](https://www.runpod.io/) 
Code into a telegram bot
Use lora such as Flux, Facefusion etc
I currently work with devs that have poor work ethic but that are really performing.
For now we're able to generate thousands of images per day fully AI generated and faceswap videos hundreds per day.
Any devs here capable of doing the same  ?

**Reply by ktechhydle** [reply]
Hello tristanvandevenne,

We appreciate your asking, however this is a contact section for our software, MPRUN. You can ask questions, leave comments, or get help from developers here.
